src/map/map.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+</head>
+<link rel="stylesheet" href="map.css">
+<body>
+  <header>
+    <h1>Closet Quest</h1>
+    <section id="user-info">
+
+      <img id="style-choice" src="assets.style-choice" alt="style choice">
+      <span id="username"></span>
+      <span id="cash">Cash</span>
+      <span id="fauxpas">Faux Pas</span>
+    </section>
+  </header>
+
+  <main>
+    <h2>Get Dressed!</h2>
+    <img src="../assests/emptyclosetquest.png">
+  </main>
+  <nav id="quest">
+    <a class="quest" href="quest.html?id=//" style="top"></a>
```

Comment:
Your linter is causing issues here because this style attribute `style="top"` isn't actually doing anything.  I would suggest targeting this a tag using CSS and the class you have or add an ID.

Reply:
Getting rid of this style tag would fix your linting issues

Reply:
Also looks like this href doesn't currently link to any page.